config/feature_flags.yml

```diff
@@ -0,0 +1,6 @@
+feature_flags:
+  omniauth_test_mode:
+    local: true
```

Comment:
The other app does something handy where for local environments the value gets pulled from an env var - while in K8s-land we want to avoid env vars, having an immediate mechanism to switch features on and off without having to touch a version-controlled file we don't want to accidentally update for everyone is quite helpful.

Reply:
yup agreed, will add this in shortly :)

Reply:
I think we're missing the final step here - this file should check the env var that you've added back in and use it to set the `local` value - otherwise the env var will just be ignored.

Reply:
my bad, I have this change locally and hadn't saved the file